contributing.md

```diff
@@ -0,0 +1,146 @@
+# Contributing
+
+When contributing to this repository, please first discuss the change you wish to make via issue,
+email, or any other method with the owners of this repository before making a change. 
+
+Please note we have a code of conduct, please follow it in all your interactions with the project.
+
+## Filing an issue
+
+1. Look for a similar issue that may already describe the proposed changes.
+
+1. If none exists, create a new issue and describe the proposed changes.
+
+1. Apply any relevant labels to the issue and set the milestone to Backlog.
+
+## Pull Request Process
+
+1. Find the issue that describes the changes you are going to make.  If no issue exists
+for the changes you want to make, see above and file an issue describing the changes, then
+continue to the next step.
+
+1. Assign the issue to yourself, and change the milestone from Backlog to the currently active
+milestone.
```

Comment:
It would help to decide how we plan on organizing work.  Milestones? Projects? Both?  

I prefer we stick to *just* projects right now.   They can do what milestones do (show progress) and give us a trello like view.  I think our goals are small enough that we don't need to have to use both.  The contributing docs should point this out. 

If we have a really huge project, then we can break it down into milestones.  Though I'm not quite sure how elegant that is with GH.

Reply:
@mostlygeek I've been using Projects to track large bodies of work and Milestones to track releases.  That's worked well for me so far.  Using milestones to track releases makes it easy to reflect to QA which issues were closed in a given release, and to historically reconstruct when things were deployed.